app/components/SocialMediaBanner.tsx

```diff
@@ -3,51 +3,87 @@ import FB from "../../public/fb.svg";
 import Image from "next/image";
 import LinkedIN from "../../public/libanner.svg";
 import Ozzie from "../../public/Ozzie Crutches SVG 2.png";
+import OzzieMobile from "../../public/Ozzie Peeking In.png";
 import TwitterX from "../../public/TwitterX.svg";
 import Yelp from "../../public/yelpbanner.svg";
 import Youtube from "../../public/youtubebanner.svg";
+import { Text1 } from "./ui/Typography/Text";
+import { Title1 } from "./ui/Typography/Title";
 
 export default function SocialMediaBanner() {
   return (
     <div className="bg-[#FCE385]">
-      <div className="grid grid-rows-2 md:gap-x-6 md:grid-flow-col py-4 px-3 justify-center">
-        <div className="row-span-3 ...">
-          <div className="flex flex-row min-h-[75%] justify-center items-center">
-            <Image src={Ozzie} alt="Ozzie" />
-          </div>
+      <div className="md:py-4 flex flex-row justify-center">
+        <DesktopBanner />
+        <MobileBanner />
+      </div>
+    </div>
+  );
+}
+
+function DesktopBanner() {
+  return (
+    <div className="hidden md:grid grid-rows-2 md:gap-x-6 md:grid-flow-col w-[45%] px-3 py-4">
+      <div className="row-span-3 ...">
```

Comment:
Add a `self-center` and remove the `...` that was just an accidental copy from tailwind's site